p2panda-net/src/engine/engine.rs

```diff
@@ -208,20 +208,18 @@ where
                 },
                 // Inform the topic discovery process and sync actor about a major network
                 // interface change.
-                Some(is_major) = interface_change_rx.recv() => {
+                Some(true) = interface_change_rx.recv() => {
```

Comment:
This change isn't directly related to the PR but I noticed it while working on the network monitor code.